docs/idf/adc.rst

```diff
@@ -0,0 +1,234 @@
+Analog to Digital Converter
+===========================
+
+Tổng quan
+--------
+
+ESP32 tích hợp 12-bit SAR ("Successive Approximation Register") ADCs (Analog to Digital Converters) và hỗ trợ đo trên 18 chân analog. Một trong số các chân này được dùng để xây dựng bộ khuyếch đại có độ lợi khả trình (programmable gain amplifier) nhằm đo các tín hiệu analog yếu.
```

Comment:
Đổi `ESP32 tích hợp 12-bit SAR ("Successive Approximation Register") ADCs (Analog to Digital Converters) ` thành `ESP32 tích hợp 12-bit SAR (Successive Approximation Register) ADCs ` và thêm footnote giải thích SAR là gì luôn